pages/common/airmon-ng.md

```diff
@@ -0,0 +1,20 @@
+# airmon-ng
+
+> Activate monitor mode on your wireless network devices.
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Activate monitor mode on wireless network devices.
```